pgx-tests/src/tests/schema_tests.rs

```diff
@@ -4,14 +4,66 @@ use pgx::*;
 
 #[pgx::pg_schema]
 mod test_schema {
+    use pgx::datum::sql_entity_graph::{PgxSql, SqlGraphEntity};
     use pgx::*;
     use serde::{Deserialize, Serialize};
 
     #[pg_extern]
     fn func_in_diff_schema() {}
 
+    #[pg_extern(sql = false)]
+    fn func_elided_from_schema() {}
+
+    #[pg_extern(sql = "generate_function")]
```

Comment:
It should actually be valid to do like `#[pg_extern(sql = generate_function)]` which would remove any ambiguity about if it's SQL or a function...

Reply:
Good point, I think that makes more sense, I'll adjust it accordingly

Reply:
@Hoverbear I just realized that the reason I used a string literal here, is that `MetaNameValue` expects the value part of `key = value` to be a `syn::Lit`. Your suggestion isn't supported AFAIK, unless you choose to avoid `Attribute::parse_meta` for parsing the attribute contents

Reply:
Hmmm I've seen this before in https://docs.rs/tracing/latest/tracing/attr.instrument.html#examples-1 ...

Reply:
I guess part of my concern (and want for that distinction) is I could very well see a bug where this code:

https://github.com/zombodb/pgx/blob/a9d18c59cf0e3932f7cf163cc2ec36318f271ad2/pgx-utils/src/sql_entity_graph/to_sql.rs#L78-L92

Parses some SQL as a Rust Path for some reason, and creates weird behavior.

Reply:
Yeah in that case, they are abandoning `Attribute::parse_meta` in favor of `Attribute::parse_args`, which allows them to use their own grammer for the attribute contents by manually parsing them. It's more effort but worth it when you need to support something that doesn't fit into the standard structured meta arguments approach. We can do something similar if you think it's worth it.

Reply:
I would definitely prefer a lack of ambiguity that if you have time, I know I've already asked a lot of you so if you are pressed on time I can wrap it up.

Reply:
I should have a commit that implements this ready shortly, just FYI